src/app/treasury/TreasurySection.tsx

```diff
@@ -27,7 +27,12 @@ export const TreasurySection = () => {
           <MetricsCard
             key={`${contract.name}-RBTC`}
             title={`${contract.name} RBTC`}
-            amount={`${buckets[index]?.RBTC?.amount ? toFixed(buckets[index].RBTC.amount) : 0}`}
+            // If the amount is less than 1e-7, show 0, this is because,
+            // toFixed may not be correctly formatting it to a clean
+            // output due to limitations in JavaScript's handling of floating-point numbers
+            amount={`${
+              Number(buckets[index]?.RBTC?.amount) < 1e-7 ? 0 : toFixed(Number(buckets[index]?.RBTC?.amount))
```

Comment:
That's weird, `toFixed()` should handle those scenarios. Plus, if `buckets[index]?.RBTC?.amount` is `undefined`, then it will throw an error. I suggest to take a look at `toFixed()` function because it seems there is an issue there.

Reply:
I verified, `toFixed` can only handle figures within this range `1e-7`. Hence why I went for this implementation